### DISCUSSION TITLE
Content of the site was partially missing

### DISCUSSION THREAD

**Discussion by myshov** [mdn/mdn-community#346] [Platform] [answered]
Hey all!

Users of Russian part of MDN a week ago opened two issues with problem about missing pages in the "Learn"-section for beginners In the issue https://github.com/mdn/translated-content/issues/12038 there are screenshots with missing pages. Now everything is working good and there is no missing content. What do you think? Was it some issue with an infrastracture? Can we prevent similar issues in the future?

https://github.com/mdn/translated-content/issues/12038
https://github.com/mdn/translated-content/issues/12029

**Comment by yin1999** [comment, marked as answer]
> Was it some issue with an infrastracture?

Yes, it has been fixed in mdn/yari#8371.

**Reply by myshov** [reply]
@yin1999 thanks for the fast reply!
Do you know there are any plans to make the process of deployment more reliable?

**Reply by yin1999** [reply]
Actually, there are many ci tests in [mdn/yari](https://github.com/mdn/yari/tree/main/.github/workflows). But for many [macros](https://github.com/mdn/yari/tree/main/kumascript/macros), they are not covered by [macro tests](https://github.com/mdn/yari/tree/main/kumascript/tests/macros). To made the process of deployment more reliable, we may need to add more tests for those macros. As for the previous fixed issue, we may add a test that the subpages' title is containing html tags.
